.github/workflows/helm-release.yaml

```diff
@@ -72,29 +73,46 @@ jobs:
         with:
           ref: ${{ inputs.ref }}
 
-      - name: Set up python
-        uses: actions/setup-python@v2
-        with:
-          python-version: "${{ inputs.python-version }}"
-
       - name: Bump version
         id: release-tag
-        uses: bakdata/ci-templates/actions/bump-version@v1.0.0
+        uses: bakdata/ci-templates/actions/bump-version@v1.5.2
         with:
           release-type: ${{ inputs.release-type }}
+          working-directory: ${{ inputs.charts-dir }}
+
+      - name: Helm package
+        uses: bakdata/ci-templates/actions/helm-package@v1.6.0
+        with:
+          charts-dir: ${{ inputs.charts-dir }}
```

Comment:
With the changes suggested above in mind, this could be the temporary directory for the gh-pages artifact (or an empty directory if there is no artifact yet).
This would also mean that you no longer include the source of the chart (`Chart.yaml`, `templates/`, `bumpversion.cfg`...).